src/libinjector/linux_injector.c

```diff
@@ -637,6 +637,10 @@ static event_response_t linux_injector_int3_cb(drakvuf_t drakvuf, drakvuf_trap_i
     return 0;
 }
 
+static bool is_interrupted(drakvuf_t drakvuf, void* data __attribute__((unused)))
```

Comment:
C++ already has a feature in the language, no need to use implementation dependent stuff.
```suggestion
static bool is_interrupted(drakvuf_t drakvuf, [[maybe_unused]] void* data)
```

Reply:
Moreover, if you want to `[[maybe_unused]]` here, then please add it to other `is_interrupted` implementations as well. Keep code consistent.

Reply:
@BonusPlay, but this is a C file. I get:
```
linux_injector.c:640:47: error: expected declaration specifiers or ‘...’ before ‘[’ token
 static bool is_interrupted(drakvuf_t drakvuf, [[maybe_unused]] void* data)
```

Reply:
I even can't omit name like in `drakvuf.cpp`:
```
linux_injector.c: In function ‘is_interrupted’:
linux_injector.c:640:1: error: parameter name omitted
 static bool is_interrupted(drakvuf_t drakvuf, void*)
```

Reply:
Hmmm, I see, in C this may be an issue, but it should work in C++ file. Interesting. Well, I guess it's better to be consistent and use same construction in both versions.

Reply:
If You would like I could use this receipt: https://stackoverflow.com/a/3599170

This is already in our source code https://github.com/tklengyel/drakvuf/blob/156321023def8a03304f441c3048efa65229f00b/src/libdrakvuf/private.h#L144

Reply:
See latest commit